lib/__tests__/defaultSeverity.test.js

```diff
@@ -2,19 +2,20 @@
 
 const postcssPlugin = require('../postcssPlugin');
 
-it('`defaultSeverity` option set to warning', () => {
+it('`defaultSeverity` option set to warning', async () => {
 	const config = {
 		defaultSeverity: 'warning',
 		rules: {
 			'block-no-empty': true,
 		},
 	};
 
-	return postcssPlugin.process('a {}', { from: undefined }, config).then((result) => {
-		const warnings = result.warnings();
+	const result = await postcssPlugin.process('a {}', { from: undefined }, config);
 
-		expect(warnings).toHaveLength(1);
-		expect(warnings[0].text).toContain('block-no-empty');
-		expect(warnings[0].severity).toBe('warning');
-	});
+	expect(result.warnings()).toMatchObject([
```

Comment:
[note] It is needed to call `result.warnings()`, so Jest `.resolves` is hard to use here.